### PR TITLE
System log event handling

### PR DESCRIPTION
**Pull request by adamstruck**

### CODE REVIEW

**Comment by buchanae** [comment]
closes #344

File: webdash/task.html [hunk 1]
```diff
@@ -12,138 +12,149 @@ <h5 style="text-align: center">Not Found</h5>
   </div>
 
   <div ng-show="loaded">
-  <table class="task-desc-table">
```
Comment (buchanae) [inline webdash/task.html]: What happened here? I thought you were just adding a section?

File: webdash/style.scss [hunk 2]
```diff
@@ -213,3 +208,8 @@ main h6 {
     border-color: #FF3D00;
   }
 }
+
+.btn-cancel-simple {
+  @extend .btn-cancel;
+  margin-left: 20px;
+}
```
Comment (buchanae) [inline webdash/style.scss]: I kept that button small on purpose. You didn't like it?

File: webdash/task.html (hunk 1 above)
Comment (adamstruck) [reply webdash/task.html]: The indentation was off in the file which is why the change looks so large.

File: webdash/style.scss (hunk 2 above)
Comment (adamstruck) [reply webdash/style.scss]: Actually the size was better.  My gripe with it was that it didn't match the style of the rest of the website.

**Comment by buchanae** [comment]
Looks great, nice work!

**Pull request merged**